src/app/components/MediaLoader/index.tsx

```diff
@@ -162,9 +175,17 @@ type Props = {
   blocks: MediaBlock[];
   className?: string;
   embedded?: boolean;
+  placeholderOverride?: boolean;
+  uniqueId?: string;
 };
 
-const MediaLoader = ({ blocks, className, embedded }: Props) => {
+const MediaLoader = ({
+  blocks,
+  className,
+  embedded,
+  placeholderOverride = true,
```

Comment:
Do we need this prop? The media config, `liveMedia.ts`, can just return no `placeholderConfig` property and it should have the same effect.